fieldsignals/signals.py

```diff
@@ -91,20 +104,28 @@ def connect_source_signals(self, sender):
         # override in subclasses
         pass
 
-    def _make_proxy_receiver(self, receiver, sender, fields):
+    def _make_proxy_receiver(self, receiver, sender, fields, trigger_during_create):
         """
         Takes a receiver function and creates a closure around it that knows what fields
         to watch. The original receiver is called for an instance iff the value of
         at least one of the fields has changed since the last time it was called.
         """
 
-        def pr(instance, *args, **kwargs):
+        def pr(instance, *args, created=False, **kwargs):
             changed_fields = self.get_and_update_changed_fields(
-                receiver, instance, fields
+                receiver,
+                instance,
+                fields,
+                trigger_during_create=(created and trigger_during_create),
             )
-            if changed_fields:
+
+            if changed_fields or (created and trigger_during_create):
```

Comment:
Adding `(created and trigger_during_create)` here - is that redundant?

`changed_fields` will be non-empty, as `trigger_during_create` is supplied to  `get_and_update_changed_fields` above

Reply:
Ah - but not in the case that there are only deferred fields